### PR TITLE
About welcome rebrand

### LINKED ISSUE #45: Rebrand about:welcome
This issue has been created for tracking purposes around rebranding about:welcome.

### PR DESCRIPTION
**Pull request by PressJump**
Fixes #45

### CODE REVIEW

File: src/browser/components/newtab/aboutwelcome/content/aboutwelcome-html.patch [hunk 1]
```diff
@@ -0,0 +1,43 @@
+diff --git a/browser/components/newtab/aboutwelcome/content/aboutwelcome.html b/browser/components/newtab/aboutwelcome/content/aboutwelcome.html
```
Comment (trickypr) [inline src/browser/components/newtab/aboutwelcome/content/aboutwelcome-html.patch]: This should not be a patch file

File: src/browser/locales/en-US/browser/welcome.ftl [hunk 2]
```diff
@@ -17,3 +17,9 @@ welcome-dialog-theme-action = Continue
 
 welcome-dialog-search = Chose a search engine
 welcome-dialog-search-action = Done
+
+# About Welcome Page Dialog
```
Comment (trickypr) [inline src/browser/locales/en-US/browser/welcome.ftl]: It is not a dialog. This comment is garbage

**Review by trickypr — CHANGES_REQUESTED** [review]

File: src/browser/components/newtab/aboutwelcome/content/aboutwelcome-html.patch (hunk 1 above)
Comment (PressJump) [reply src/browser/components/newtab/aboutwelcome/content/aboutwelcome-html.patch]: Replaced the patch file with itself in src which can be overwritten.

File: src/browser/locales/en-US/browser/welcome.ftl (hunk 2 above)
Comment (PressJump) [reply src/browser/locales/en-US/browser/welcome.ftl]: Removed Dialog from code comment

**Review by trickypr — APPROVED** [review]

**Pull request merged**